src/views/Addresses.vue

```diff
@@ -6,26 +6,27 @@
 
         <b-form-group label-cols-sm="3" class="mb-0">
           <b-input-group>
-            <b-form-input v-model="filter" debounce="750" placeholder="Search by address"></b-form-input>
+            <b-form-input v-model="filter" placeholder="Search by address"></b-form-input>
             <b-input-group-append>
-              <b-button :disabled="!filter" @click="filter = ''" class="clear-button">Clear</b-button>
+              <b-button :disabled="!filter" @click="clearFilter" class="clear-button">Clear</b-button>
             </b-input-group-append>
           </b-input-group>
         </b-form-group>
       </b-card-header>
 
+      <!-- Table implementation for addresses -->
       <b-table responsive table-class="compact" :items="items" :fields="addresses_fields" stacked="sm"
-        :current-page="page" :per-page="per_page" :sort-by.sync="sortBy" :sort-desc.sync="sortDesc"
-        :sort-direction="sortDirection" @filtered="onFiltered">
+        :sort-by.sync="sortBy" :sort-desc.sync="sortDesc" @sort-changed="onSort">
```

Comment:
`@sort-changed="onSort"` should be removed as the watchers are already triggering the sorting.

Keeping watchers + handler is triggering the sort twice everytime.


```suggestion
        :sort-by.sync="sortBy" :sort-desc.sync="sortDesc">
```